.github/labeler.yml

```
@@ -65,52 +68,76 @@ Configs:
+      - '.coveragerc'
+      - '.gitattributes'
+      - '.gitignore'
       - requirements.txt
-      - test-requirements.txt
       - tests/requirements.txt
       - docs/requirements.txt
 
+# Add 'documentation' label to any documentation file changes
 documentation:
   - changed-files:
     - any-glob-to-any-file:
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)